ivy-test.el

```diff
@@ -241,6 +241,26 @@ will bring the behavior in line with the newer Emacsen."
                   (ivy--regex "(foo bar"))
                  "(\\(foo).*?(bar)")))
 
+(ert-deftest counsel-url-expand ()
+  "Test ffap expansion using counsel-url-expansions."
+  ;; no expansions defined
+  (let ((counsel-url-expansions nil))
```

Comment:
This comes down to a preference in style, but FYI this can also be written as `(let (counsel-url-expansions) ...)`.

Reply:
thanks, I'll switch.